compass/landice/mesh.py

```diff
@@ -102,3 +102,100 @@ def set_rectangular_geom_points_and_edges(xmin, xmax, ymin, ymax):
         dtype=jigsawpy.jigsaw_msh_t.EDGE2_t)
 
     return geom_points, geom_edges
+
+
+def set_cell_width(self, section, thk, vx=None, vy=None,
+                   distToEdge=None, distToGroundingLine=None):
+    """
+    Set cell widths based on settings in config file to pass to
+    :py:func:`mpas_tools.mesh.creation.build_mesh.build_planar_mesh()`.
+
+    Parameters
+    ----------
+    section : str
+        Section of the config file from which to read parameters
```

Comment:
Either here or above in the main text of the docstring, you will need to specify all of the config options that are required to be in the given config section for the function to work.